java/arcs/android/storage/service/BindingContext.kt

```diff
@@ -209,8 +214,10 @@ class BindingContext(
                     store().off(token)
                     resultCallback.onResult(null)
                 } catch (e: Exception) {
+                    log.debug(e) { "Exception occurred while unregistering callback" }
                     resultCallback.onResult(
-                        CrdtException("Callback unregistration failed", e).toProto().toByteArray()
+                        CrdtException("Exception occurred while unregistering callback", e)
```

Comment:
This is no good, we still lose the original exception message on the other side.